api/turing/models/ensembler.go

```diff
@@ -79,3 +89,22 @@ func (c EnsemblerDockerConfig) Value() (driver.Value, error) {
 func (c *EnsemblerDockerConfig) Scan(value interface{}) error {
 	return json.Unmarshal(value.([]byte), &c)
 }
+
+// Value implements sql.driver.Valuer interface so database tools like go-orm knows how to serialize the struct object
+// for storage in the database
+func (c EnsemblerPyFuncRefConfig) Value() (driver.Value, error) {
+	return json.Marshal(c)
+}
+
+// Scan implements sql.Scanner interface so database tools like go-orm knows how to de-serialize the struct object
+// from the database
+func (c *EnsemblerPyFuncRefConfig) Scan(value interface{}) error {
+	return json.Unmarshal(value.([]byte), &c)
+}
+
+const (
```

Comment:
Are these constants better placed in the service layer? Per the previous suggestion to move the image building logic into `api/turing/service/router_deployment_service.go`, these consts can move there too. Don't think they need to be in the `models` package.

Reply:
Oh yea that's true, or another way I can think of is to dump them in `config_dev.yaml` together with `EnsemblerServiceBuilderConfig`. Or would that be overkill?

Reply:
Haha `EnsemblerServiceBuilderConfig` is probably an overkill for now.